colander/__init__.py

```diff
@@ -349,7 +349,15 @@ def __call__(self, node, value):
         if self.match_object.match(value) is None:
             raise Invalid(node, self.msg)
 
-EMAIL_RE = "(?i)^[A-Z0-9._%!#$%&'*+-/=?^_`{|}~()]+@[A-Z0-9]+([.-][A-Z0-9]+)*\.[A-Z]{2,22}$"
+EMAIL_RE = r"""(?ix) # matches case invariant with spaces and comments ignored
```

Comment:
"invariant" should be "insensitive"

Also please append " for the entire expression" so that it is more clear that `A-Z` includes `a-z`.